### PR TITLE
Fixed MyKey LockID

### PR DESCRIPTION
**Pull request by zProAle**
# What's new

- Fixed checking if a MyKey has LockID active

# Verification 

- [ Describe how to verify changes ]

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

File: applications/main/nfc/plugins/supported_cards/mykey.c [hunk 1]
```diff
@@ -13,7 +13,7 @@ static bool mykey_is_blank(const St25tbData* data) {
 }
 
 static bool mykey_has_lockid(const St25tbData* data) {
-    return (data->blocks[5] & 0xFF) == 0x7F;
```
Comment (gornekich) [inline applications/main/nfc/plugins/supported_cards/mykey.c:16]: Can' verify it with my samples. @zProAle do you have documentation supporting this layout?
@augustozanellato can you have a look?

File: applications/main/nfc/plugins/supported_cards/mykey.c [hunk 2]
```diff
@@ -13,7 +13,7 @@ static bool mykey_is_blank(const St25tbData* data) {
...
+    return (data->blocks[5] >> 24) == 0x7F;
```
Comment (gornekich) [inline applications/main/nfc/plugins/supported_cards/mykey.c:16]: Can' verify it with my samples. @zProAle do you have documentation supporting this layout?
@augustozanellato can you have a look?

Comment (augustozanellato) [reply applications/main/nfc/plugins/supported_cards/mykey.c:16]: > Can' verify it with my samples. @zProAle do you have documentation supporting this layout? @augustozanellato can you have a look?

LGTM, I talked with @zProAle about the issue and suggested him making a PR here. 
The lockid sample I sent you had block5 flipped due to some lack of sleep.

Comment (gornekich) [reply applications/main/nfc/plugins/supported_cards/mykey.c:16]: Thanks!

**Review by gornekich — APPROVED** [review]
Thanks!

**Review by skotopes — APPROVED** [review]

**Pull request merged**